etl/steps/data/garden/wb/2025-05-12/enterprise_surveys.meta.yml

```diff
@@ -0,0 +1,189 @@
+              - Measures the percentage of firms that reported being expected to provide informal payments or gifts during tax inspections or meetings
+              - Focuses specifically on firms’ interactions with tax officials which are a known point of corruption vulnerability
+              - Data are collected through structured interviews with private sector firms using stratified random sampling
+              - Corruption in tax inspections increases operational unpredictability and creates unofficial costs
+              - Regional and global averages are computed by taking a simple average of country-level results using only the latest available survey data per country since 2014 following the Enterprise Surveys Global Methodology
+          presentation:
+            title_public: Share of firms expected to bribe tax officials
+          display:
+            name: Tax officials
+
+      percent_of_firms_expected_to_give_gifts_to_secure_government_contract:
+          title: Share of firms expected to give bribes to secure government contracts
+          description_short: Percent of firms expected to give bribes to win government contracts.
+          description_from_producer: |-
+            Percentage of firms that believe companies like theirs are typically expected to make informal payments to public officials to secure government contracts.
+          description_key:
+              - Measures the share of firms that believe companies like theirs are expected to pay informal payments or gifts to secure government contracts
+              - Captures perceived expectations rather than direct reports of bribe payments
+              - Data are collected through structured interviews with private sector firms using stratified random sampling
+              - Bribery in procurement increases business costs reduces fairness and deters competition from firms unwilling or unable to pay
+              - Regional and global averages are computed by taking a simple average of country-level results using only the latest available survey data per country since 2014 following the Enterprise Surveys Global Methodology
+          presentation:
+            title_public: Share of firms expected to bribe secure government contracts
+          display:
+            name: Government contract
+
+      percent_of_firms_expected_to_give_gifts_to_get_an_operating_license:
+          title: Share of firms expected to give bribes to obtain an operating license
+          description_short: Percent of firms expected to give bribes to get an operating license.
+          description_from_producer: |-
+            Percentage of firms reporting that they were expected to provide gifts or informal payments to obtain an operating license. Refusals are treated as affirmative.
+          description_key:
+              - Measures the percentage of firms that reported being expected to pay informal payments to obtain an operating license
```

Comment:
Maybe it's worth explaining (with a DoD or an additional point) what an operating license is.